src/main/java/com/nutomic/syncthingandroid/syncthing/RestApi.java

```diff
@@ -204,6 +204,10 @@ public void onWebGuiAvailable() {
         new GetTask() {
             @Override
             protected void onPostExecute(String s) {
+                if (s == null) {
+                    Log.w(TAG, "Empty JSON string when retrieving version information");
```

Comment:
ok, we can check again if there are still error reports coming in from this point